distribution/bwc/build.gradle

```diff
@@ -196,3 +204,25 @@ subprojects {
     }
   }
 }
+
+class IdentingOutputStream extends OutputStream {
+
+  private final OutputStream delegate
+
+  public IdentingOutputStream(OutputStream delegate) {
+      this.delegate = delegate
+  }
+
+  @Override
+  public void write(int b) {
+    write([b] as int[], 0, 1)
+  }
+
+  public void write(int[] bytes, int offset, int length) {
+    delegate.write(
+            new String(bytes, offset, length)
+                    .replace("\n", "\n  [bwc] ")
```

Comment:
I think this is going to be pretty inefficient. Instead, you can loop over the bytes looking for newline. When you find a newline, emit the array up to that point. Then emit a fixed byte array you have with the newline and prefix. Then continue the loop. You won't need to create any temporary objects.